perceval/utils/logging/config.py

```diff
@@ -43,10 +43,10 @@ class LoggerConfig(dict):
     """This class represent the logger configuration as a dictionary and can be used to save it into persistent data.
     On class initialization, the configuration will be loaded from persistent data.
     """
-    def __init__(self):
+    def __init__(self, persistent_data: PersistentData = PersistentData()):
```

Comment:
Prefer passing `None` by default and initializing the object after, here it is initialized when the method is defined, even if it's never used